content/docs/use-cases/versioning-data-and-model-files/index.md

```diff
@@ -1,130 +1,167 @@
 # Versioning Data and Model Files
```

Comment:
btw, should we just name it `Data and Models` (remove Files, I'm not sure when we added this and why? or was it always like this?)

Reply:
I was just thinking that! (reviewing #1856)

It was "Data and Model Files Versioning" before, and I changed it to this at some point.

I guess "files" only intends to refer to models (i.e. versioning data and versioning model files), but it's not clear. Will update

Reply:
Initial update in 384218da

But I need to change a bunch of links from other docs which will add like 8 files to this PRs change... You may want to review the changes so far first?